x-pack/plugins/security_solution/public/cases/components/case_view/index.tsx

```diff
@@ -5,51 +5,34 @@
  * 2.0.
```

Comment:
this file looks very different, but it's not what it seems. I deleted the original `case_view/index.tsx` and renamed `case_view/case_view_from_plugin.tsx` to `case_view/index.tsx`